README.md

```diff
@@ -169,3 +169,9 @@ Compares experiment settings accros development/staging/production environments.
 
     biomage experiment compare my-experiment-id
 
+#### experiment copy
+
+Copies one experiment information from a given env to another. See `biomage experiment copy --help` for more details.
+
+`biomage experiment copy e52b39624588791a7889e39c617f669e default`
```

Comment:
how do you specify from which environment to copy from which and what is being done by default? For instance, after running the example command `biomage experiment copy e52b39624588791a7889e39c617f669e default`, what will be copied from where to where?

Reply:
I actually started designing this commands to be close to git like `git pull origin master` but I think they are starting to have too many parameters so I should switch them to use options like `biomage experiment copy --experiment_id= e52b39624588791a7889e39c617f669e --sandbox_id=default`. What do you think?

Reply:
This sounds like a good idea, yes

Reply:
done